tests/unit/Actions.spec.ts

```diff
@@ -72,25 +75,25 @@ describe('Actions component', () => {
     const localVue = createLocalVue()
     localVue.use(VueRouter)
     const router = mockRouter.mock()
-    wrapper = shallowMount(Actions, { localVue, store, router, vuetify })
+    wrapper = shallowMount(Actions, { localVue, router, vuetify })
   })
 
   afterEach(() => {
     wrapper.destroy()
   })
 
   it('Enables File and Pay button when certify from is valid', async () => {
-    store.state.stateModel.certifyState = {
+    store.stateModel.certifyState = {
       valid: true,
       certifiedBy: 'Some certifier'
     }
-    store.state.stateModel.entityType = 'BEN'
-    store.state.stateModel.nameRequest = { entityType: 'BEN' }
-    store.state.stateModel.defineCompanyStep = { valid: true }
-    store.state.stateModel.addPeopleAndRoleStep = { valid: true }
-    store.state.stateModel.createShareStructureStep = { valid: true }
-    store.state.stateModel.incorporationAgreementStep = { valid: true }
-    store.state.stateModel.effectiveDateTime = { valid: true }
+    store.stateModel.entityType = CorpTypeCd.BENEFIT_COMPANY
+    store.stateModel.nameRequest.legalType = CorpTypeCd.BENEFIT_COMPANY
```

Comment:
Thanks for using enums here.

Line 91 seems like a big change. Why wasn't it broken before?

Reply:
I was wondering the same. I changed it because of TS complaint
![image](https://user-images.githubusercontent.com/116035339/236584662-64448a0f-5761-489f-8b93-6bc7051883a4.png)

NameRequest interface:
![image](https://user-images.githubusercontent.com/116035339/236584800-68e0a810-9b20-48ed-bec3-b617b6c19551.png)

Reply:
Hooray for Typescript and typings :)

Now I'm wondering how the unit test worked before !? That's a bit out of scope for this ticket but it goes to show that our tests aren't perfect.

Reply:
As long as the unit test passes now, I'm OK with this.